documentation/pelicanconf.py

```diff
@@ -8,6 +8,14 @@
 SITESUBTITLE = "The Best Pelican Theme"
 SITEURL = ""
 
+AUTHORS = {
```

Comment:
@iranzo you should add your entry to the existing dictionary that has my entry in it too.

Reply:
yup, done in next push